Automattic/jetpack
# CLI: Automatically Add Changelog Files to Git Staging

Fixes #19426

#### Changes proposed in this Pull Request:

* Allows `jetpack changelog add` to also add new changelog files to be tracked by git automatically, making the commit process smoother so you don't have to `git add` after.

This PR utilizes the [Simple Git package](https://www.npmjs.com/package/simple-git), which lets node easily handle running git commands. I could see this being helpful if we want to extend this to handle automattic commits, as well as checking project files for updates to prompt the changelogger.

Open to alternatives if there's another way of working with this!

#### Jetpack product discussion


N/a 
#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Use `jetpack changelog add` to add a new changelog file.
* After successfully adding a changelog file, run `git status` 
* You should see the newly added changelog file staged already.
